lib/general.sh

```diff
@@ -1155,6 +1160,8 @@ prepare_host()
 
 	if [[ ${#deps[@]} -gt 0 ]]; then
 		display_alert "Installing build dependencies"
+		# don't prompt for apt cacher selection
+		sudo echo "apt-cacher-ng    apt-cacher-ng/tunnelenable      boolean false" | sudo debconf-set-selections
 		apt-get -q update
 		apt-get -y upgrade
```

Comment:
Since you touched this part of the code; this apt-get upgrade has bit me more than once. Sometimes I'm just looking to build Armbian, not do a hundred-package upgrade of my desktop. I understand why it might be useful...

Reply:
Docker / VM ;)